sdk/confidentialledger/azure-security-confidentialledger/README.md

```diff
@@ -55,34 +55,7 @@ To use the [DefaultAzureCredential][DefaultAzureCredential] provider shown below
 
 Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
 
-##### Example
```

Comment:
do we have replacement for this delete example?

Reply:
I will replace it!